Content/1.1.4 - Interaction Review/interactionReview.json

```
@@ -30,5 +30,9 @@
 	"interactionReview_depositExceptions_depositorChangeAdd": "Add Depositor",
 	"interactionReview_depositExceptions_depositorChangeRemove": "Remove Depositor",
 	"interactionReview_hiddenAsset_deposit": "This asset is hidden and will not be visible in your Account",
-	"interactionReview_hiddenAsset_withdraw": "This asset is hidden and is not visible in your Account"
+	"interactionReview_hiddenAsset_withdraw": "This asset is hidden and is not visible in your Account",
+	"interactionReview_unknown_deposits": "Unknown deposits to this account may occur",
+	"interactionReview_unknown_amount": "Amount of deposit is unknown",
+	"interactionReview_at_least": "At least",
+	"interactionReview_no_more_than": "No more than"
```

We should use the underscore to catrgorize, and use `camelCase` rather than `snake_case` for each of the potential 3 levels of categories. This means, it should look as this:
```suggestion
	"interactionReview_atLeast": "At least",
	"interactionReview_noMoreThan": "No more than"
```

The 
```
interactionReview_unknown_deposits
interactionReview_unknown_amount
```
could also be changed into
```
interactionReview_unknownDeposits
interactionReview_unknownAmount
```
if you wanted to, but I guess taking `unknown` as a category is fine (same as we do for `hiddenAssets`.